docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,196 @@
+Installing and Using the XRootD Client 
+==============================================
 
-Install XRootd Client 
-=====================
+XRootD is a high performance data system widely used by several science VOs on OSG to store and to distribute data to
+jobs.
+It can be used to create a data store from distributed data nodes or to serve data to systems using a distributed
+caching architecture.
+Either mode of operation requires you to install the XRootD client software.
 
-XRootD is a high performance network storage system widely used in high energy physics experiments such as ATLAS and ALICE. The underlying XRootD data transfer protocol provides highly efficient access to ROOT based data files.
+This page provides instructions for installing and using the XRootD client software to access data using XRootD.
 
-This page provides instructions for using a XRootD client to connect to a XRootD storage system.
+As a user you have three different ways to interact with XRootD: 
 
-Installing a XRootD Client
---------------------------
+1. Using the XRootD client tools command 
+1. Using LD\_PRELOAD to use XRootD libraries with Unix tools
+1. Using a XRootDFS FUSE mount to mount a local XRootD data store
 
-As a user you have three different ways to interact with XRootd: a file system mounted with XRootDFS, the application `xrdcp`, LD\_PRELOAD.
+We'll show how to install the XRootD client software and use all three mechanisms to access data.
 
-To use a XRootD file system mounted with XRootDFS you don't need any special setup, you will need the XRootD FUSE module.
+!!! Note
+    Only the client tools method should be used to access XRootD systems across a WAN link.
 
-To use `xrdcp` you need to have it in your path (e.g. `source $INSTALL_DIR/setup.sh`, where `$INSTALL_DIR` is the XRootD installation directory)
 
-The use of LD\_PRELOAD is not recommended if you have any alternative available.
+Before Starting
+---------------
 
-### Install Repositories 
+Before starting the installation process, consider the following points *if you are using a FUSE mount to access
+XRootD:*
 
-If you have not done so, you will need to install the [proper yum repositories](../common/yum):
+-   **User IDs:** If it does not exist already, you will need to create Linux users `xrootd` 
 
+As with all OSG software installations, there are some one-time (per host) steps to prepare in advance:
 
-### Installing XRootD Client
+- Ensure the host has [a supported operating system](/docs/release/supported_platforms)
```

Comment:
The `docs/` dir in the repo is the root dir that's used for documents so your links shouldn't start with `/docs`

Reply:
In mkdocs, site-relative links are relative to the `docs/` folder in the repo by default. So with your example, the format would look like `[link name](/data/install-xrootd#optional-enabling-a-fuse-mount)`

Reply:
Okay, in the docker container I've been using, the /data/ links are broken but the /docs/data links aren't.  Updated this and pushed the fixes